packages/frontend-plugin-api/src/index.ts

```diff
@@ -22,7 +22,10 @@
 
 export * from './apis';
 export * from './components';
+export * from './definitions';
 export * from './extensions';
+export * from './icons';
 export * from './routing';
 export * from './schema';
+export * from './system';
```

Comment:
I think it's good to keep the `apis/{definitions,system}` split tbh? worried that this is gonna be a bit confusing.